analyzers/src/SonarAnalyzer.CSharp/SymbolicExecution/CSharp/CSharpExplodedGraph.cs

```diff
@@ -649,7 +649,7 @@ private ProgramState VisitVariableDesignation(VariableDesignationSyntaxWrapper v
             {
                 // Push value for the discard, it will be popped when visiting the block for the
                 // corresponding case statement.
-                newProgramState = newProgramState.PushValue(SymbolicValue.Create());
+                newProgramState = newProgramState.PushValue(new SymbolicValue());
```

Comment:
This has no functional effect. It's just to use same API as below (because we don't need the logic inside `.Create()` here).